.husky/pre-commit

```diff
@@ -2,5 +2,7 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npm run pre-commit-translate
+npm run pre-commit-trustpilot
```

Comment:
I think it should be not a `pre-commit` thing, but a `build` or some type of `pre-build` step. Otherwise, each developer will introduce changes for TrustPilot  but it will not relate to their PRs. 

Also, commit and actual deploy can differ in dates and we will not get the most up to date score.